scripts/install/installer

```diff
@@ -640,19 +640,25 @@ class JumpScale7(Node):
         self.log("Start JumpScale services")
         self.remote.run("systemctl start {}".format(" ".join(self.get_services())))
 
-    def stop(self):
+    def stop(self, ays=False):
         self.log("Stop JumpScale services")
-        res = self.remote.run("which ays", check=False)
-        if res.exit_status:
+        if ays:
             self.remote.run("ays stop")
+            self.remote.run('rm -f /etc/systemd/system/jsagent_main.service')
```

Comment:
before this you want to disable it aswel
systemctl disable jsagent_main ays
and remove ays aswell while you are at ti